src/installer/dbinstall_mysql_lang.sql

```diff
@@ -19,8 +19,8 @@ CREATE TABLE `DBPREFIXlanguages` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 
 INSERT INTO `DBPREFIXlanguages` (`langid`, `englishname`, `nativename`, `langcode`, `isdefault`) VALUES
-(1, 'English', 'English', 'en', 1),
-(2, 'English (US)', 'English (US)', 'en-us', 0),
+(1, 'English', 'English', 'en-gb', 1),
```

Comment:
Again: this is not the language used in day.js
The language for day.js is pulled from `DAYJS_LANG` variable set in every language.

Reply:
Shouldn't we set this accordingly like the `DAYJS_LANG`? That is what I thought about that change. Anyway, I will remove it for now

Reply:
As far as I remember, those are the language codes from our translation platform.